crates/cargo-bsp/src/project_model/sources.rs

```diff
@@ -241,57 +257,66 @@ mod tests {
 
         use super::*;
 
+        struct TestDirInfo {
+            _root_temp_dir: TempDir,
```

Comment:
może komentarz że to musi być trzymane wewnątrz struktury bo inaczej się ten tempdir usunie przy dropie